api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminControllerTest.java

```diff
@@ -157,6 +158,31 @@ public void getFederatedWorkspaceDetails_404sWhenNotFound() {
     assertThat(response.getStatusCodeValue()).isEqualTo(404);
   }
 
+  @Test
+  public void getCloudStorageTraffic_sortsPointsByTimestamp() {
+    TimeSeries timeSeries =
```

Comment:
oh god is this really how spotless wants it

Reply:
If it is, could you at least put a full blank line between statements for readability